DirectEmployers/MyJobs
# [PD-3408] Create Test for Loading Reducer in Analytics

## What's this PR do?
PR creates the test specs for the loading reducer inside of analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Code Review, or inside of the myjobs folder run dkgg watch to pull up the watching of the test changes

## How should this be manually tested?
Pull down the repository. Inside of the myjobs folder run dkgg watch and make sure all of the specs pass for the test. Also you can view the code created

## Any background context you want to provide?
